modules/ROOT/pages/faq.adoc

```diff
@@ -233,3 +233,9 @@ systemd:
     - name: docker.service
       mask: true
 ----
+
+== Are Fedora CoreOS x86_64 disk images hybrid BIOS+UEFI bootable?
+
+The x86_64 images we provide can be used for either BIOS (legacy) boot or UEFI boot. They contain a hybrid BIOS/UEFI partition setup that allows them to be used for either. The exception to that is the `metal4k` 4k native image, which is targeted at disks with 4k sectors and https://github.com/coreos/coreos-assembler/blob/12029fea7798fa5d3535eafcf8c3d02f9a6095e4/src/cmd-buildextend-metal#L200-L202[does not have a BIOS boot partition] because 4k native disks are https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/hard-drives-and-partitions#advanced-format-drives[only supported with UEFI].
```

Comment:
Maybe for the first sentence we can add the words "bare metal" since that's how it's labeled in the download page, e.g.:

```
The x86_64 bare metal images we provide can be used for either BIOS (legacy) boot or UEFI boot.
```

?

Reply:
I think we should keep it generic. I would have claimed otherwise, but the truth is that this isn't limited to our bare metal images. From looking at the code the only x86_64 image we create that isn't hybrid is the 4kn disk image. Am I reading it correctly?

Reply:
Sure, that's true. Though OTOH it's not like it matters much for most other platforms, and although I love the consistency, it's possible we may change it in the future (see e.g. https://github.com/coreos/fedora-coreos-config/pull/407) and should reverse the right to break that.

No strong opinion either way though.

Reply:
Thanks. Let's merge as-is then.